### ISSUE TITLE
Anchor - Unknown browser console error

### ISSUE THREAD

**Issue by fendyjong** [grommet/grommet#1667]
I am using Anchor to open PDF in a new tab. But every time I click on the Anchor, it always shows an unknown error in the browser console.

`Uncaught TypeError: Cannot read property 'apply' of undefined`

I have search through all of my codes and I am not finding any variable or property called 'apply'

### Code
```
<Anchor href={url}
               target='blank'
               icon={<DownloadIcon />}
               label='Download PDF'
               disabled={!url} />
```

### Expected Behavior
No error

### Actual Behavior
Unknown browser console error

![screen shot 2017-10-05 at 3 26 19 am](https://user-images.githubusercontent.com/30851388/31198416-670de4b8-a97e-11e7-9e93-80096cf342f9.png)

### Your Environment
* Grommet version: grommet @1.7.0
* Browser Name and version: Chrome Version 61.0.3163.100 (Official Build) (64-bit)
* Operating System and version (desktop or mobile): macOS Sierra Version 10.12.6

**Comment by karatechops** [comment]
Hey @fendyjong, this was fixed in 1.8.0. Can you upgrade your app and confirm?

**Comment by fendyjong** [comment]
Hi @karatechops, I confirm that this issue is fixed in 1.8.0
